hapi-fees/src/main/java/com/hedera/node/app/hapi/fees/calc/OverflowCheckingCalc.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2021-2022 Hedera Hashgraph, LLC
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
```

Comment:
I assume this change was done by Spotless, which I find concerning as this is simply wrong. The file was created in 2021 and the copyright notice has to reflect that. 

IANAL, but my understanding is, that these headers are not mandatory. Therefore having no header is better than having one with wrong data. We should discuss this.